spec/spec_params.rb

```diff
@@ -657,6 +657,6 @@ module OptimizelySpec
   VALID_CONFIG_BODY_JSON = JSON.dump(VALID_CONFIG_BODY)
 
   INVALID_CONFIG_BODY = VALID_CONFIG_BODY.dup
-  INVALID_CONFIG_BODY['version'] = '1'
+  INVALID_CONFIG_BODY['version'] = '5'
```

Comment:
Leave testcase for version '1' as it is. 
Add a new test case for 5.

Reply:
No need, since we only checking supported versions.